packages/devtools/src/datamodel.ts

```diff
@@ -188,11 +208,36 @@ export class ModelManager {
   async loadStream(streamID: StreamRef | string): Promise<TileDocument> {
     const id = typeof streamID === 'string' ? streamID : streamID.baseID.toString()
     if (this.#streams[id] == null) {
-      this.#streams[id] = TileDocument.load(this.#ceramic, id)
+      this.#streams[id] = this._loadAndValidateStream(id)
     }
     return await this.#streams[id]
   }
 
+  /** internal */
+  async _loadAndValidateStream(id: string): Promise<TileDocument> {
+    const stream = await TileDocument.load<Record<string, any>>(this.#ceramic, id)
+    if (stream.anchorCommitIds.length > 0) {
+      throw new Error(`Invalid stream ${id}: contains anchor commit`)
+    }
```

Comment:
Did you confirm that the current IDX models doesn't have anchors on them?

Reply:
Yes I checked them on tiles.ceramic.community, they don't have anchors 👍